tailwind.config.js

```diff
@@ -38,10 +38,13 @@ module.exports = {
       },
       flex: {
         '1-0-0': '1 0 0%',
+      },
+      screens: {
+        'sm': {
+          max: '430px'
+        }
       }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a new screen size 'sm' with a max width of 430px seems fine. However, please ensure that this new breakpoint doesn't conflict with the existing breakpoints in your Tailwind CSS configuration. If there are other places in your code where you're using the 'sm' breakpoint, those might be affected by this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->